src/MCPClient/osdeps/Ubuntu-18.json

```diff
@@ -11,7 +11,7 @@
   ],
   "osdeps_repos": [
     "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu-externals xenial main",
```

Comment:
Why is this needed?

Reply:
Some dependencies have not been rebuilt for 18.04, as the packages from 16.04 work without problems. I would prefer to leave 1.7 as is, and improve the repository management for 1.8